### ISSUE TITLE
--combine Feature Not Working

### ISSUE THREAD

**Issue by alexgaganashvili** [open-policy-agent/conftest#414]
Hi,

The following example with the --combine flag does not seem to be working as expected:
https://www.conftest.dev/options/

The output shows 0 failures and that the test has passed.

Thanks,
Alex

**Comment by jpreese** [comment]
The docs were updated to reflect the new `--combine` behavior which hasn't been released yet, but will be in `0.22.0`.

The previous docs for `0.21.0` that show the combine behavior can be found [here](https://github.com/open-policy-agent/conftest/blob/5d1e3d8ae8c6071079c69e88f2f70db65c50c5d2/docs/options.md)

```rego
package main


deny[msg] {
  deployment := input["deployment.yaml"]["spec"]["selector"]["matchLabels"]["app"]
  service := input["service.yaml"]["spec"]["selector"]["app"]

  deployment != service

  msg = sprintf("Expected these values to be the same but received %v for deployment and %v for service", [deployment, service])
}
```

**Comment by alexgaganashvili** [comment]
Got it. Thanks. Yeah, I figured I had to use yaml file name as the key when I was troubleshooting the issue, which was not what was being demonstrated in the example. 
I'll wait for the release of 0.22.0.
